### PR TITLE
Fix footer sponsors section

### PR DESCRIPTION
**Pull request by luciagirasoles**
Update the Sponsors area in the footer to fix its appearance and ensure it is up to date.

> Screen Shot 2023-05-16 at 10 46 25 PM

Solution: Update footer sponsors section to more closely match rubygems.org, as outlined here:
https://github.com/rubygems/rubygems.org/pull/3604
https://github.com/rubygems/rubygems.org/pull/3610

With this PR:
Desktop
> Screenshot 2023-11-11 at 1 38 43 AM

Mobile
> Screenshot 2023-11-11 at 1 31 30 AM

### CODE REVIEW

File: stylesheets/modules/_footer.scss [hunk 1]
```diff
@@ -48,32 +40,17 @@
...
-    right: auto;
-    left: auto;
-  }
-  @media (max-width: 579px) {
-    width: 78%;
-  }
-  @media (min-width: 580px) and (max-width: 1039px) {
-    width: 90%;
-  }
+	margin: 0 2%;
```
Comment (jenshenny) [inline stylesheets/modules/_footer.scss:46]: It looks like the tabbing is odd in some areas as the default is 2 spaces but most of changes have 4 spaces as a tab (eg. Line 53, Line 173-196).

File: stylesheets/application.css
Comment (jenshenny) [inline stylesheets/application.css:1]: It's difficult to view the actual changes in this file, can you be able to remove the formatting changes?

**Review by jenshenny — COMMENTED** [review]
Thanks @luciagirasoles! Other than the formatting on the CSS, it looks good to me

**Review by jenshenny — COMMENTED** [review]
Thanks @luciagirasoles! Other than the formatting on the CSS, it looks good to me

**Comment by luciagirasoles** [comment]
Hi. I'm also adding updates from [rubygems.org:3996](https://github.com/rubygems/rubygems.org/pull/3996)

File: stylesheets/modules/_footer.scss (hunk 1 above)
Comment (luciagirasoles) [reply stylesheets/modules/_footer.scss]: thank you, I could fix it

File: stylesheets/application.css
Comment (luciagirasoles) [reply stylesheets/application.css:1]: Sure

**Review by jenshenny — APPROVED** [review]

**Comment by hsbt** [comment]
FYI: I reverted build artifact with RG 3.3 at https://github.com/rubygems/guides/commit/3667c39aea8d5a0a86dac3040020a0bd5e5b2999

**Pull request merged**